src/server/app.py

```diff
@@ -1,11 +1,48 @@
-from flask import Flask
+import json
+
```

Comment:
I had to import sys and append src to it to make this run. There should be some README that explains you need to reference src as a module.